DSharpPlus/Net/Models/GuildEditModel.cs

```diff
@@ -103,6 +104,31 @@ public class GuildEditModel : BaseEditModel
         /// </summary>
         public Optional<string> PreferredLocale { internal get; set; }
 
+        /// <summary>
+        /// The new description of the guild
+        /// </summary>
+        public Optional<string> Description { get; set; }
+
+        /// <summary>
+        /// The new discorvery splash image of the guild
+        /// </summary>
+        public Optional<string> DiscoverySplash { get; set; }
+
+        /// <summary>
+        /// A list of <see href="https://discord.com/developers/docs/resources/guild#guild-object-guild-features">guild features</see>
+        /// </summary>
+        public Optional<IEnumerable<string>> Features { get; set; }
```

Comment:
Features can't be edited

Reply:
that is one i couldnt test - those are set by discord i dont even know why they should be editable, i just put them in there because they are in the docs

Reply:
Yep, they are in the API fields for Modify Guild. Whether or not the API will accept them like what @Lulalaby said is an entirely differently matter. We'll probably ask a staff member for clarification later:tm:.

Reply:
> Features can't be edited


> Yep, they are in the API fields for Modify Guild. Whether or not the API will accept them like what @Lulalaby said is an entirely differently matter. We'll probably ask a staff member for clarification later™️.


Community, Discoverability, Threads, etc are all features that can be edited.

Reply:
Now what exactly can be edited by bots is something else.

Reply:
I modified my guild features with a bot. I had to fix everything via client afterwards. I suggest to remove this entirely.
It's not a good idea to do it via bot.

Reply:
> 
> 
> I modified my guild features with a bot. I had to fix everything via client afterwards. I suggest to remove this entirely.
> It's not a good idea to do it via bot.

In this case should the field be kept on the payload, but not exposed via methods, or just outright removed?

Reply:
I would suggest to remove it, but @Neuheit should decide.

Reply:
any decision on this yet?
@Neuheit @Lulalaby

Reply:
I don't think so

Reply:
If it's exposed by the API it should be kept.